packages/core-components/src/components/AlertDisplay/AlertDisplay.tsx

```diff
@@ -42,7 +42,7 @@ export type AlertDisplayProps = {
 
 /** @public */
 export function AlertDisplay(props: AlertDisplayProps) {
-  const [messages, setMessages] = useState<Array<AlertMessage>>([]);
+  const [messages, setMessages] = useState<Array<Partial<AlertMessage>>>([]);
```

Comment:
Feels a bit overkill? 😅

Reply:
Yeah I was just thinking, it'd make it more clear that perhaps required properties aren't actually there, for future logic... So we don't end up in the same situation again.

Reply:
I would be slightly confused to Partial here tbh. I'd say skip it for now and go with the other tweak 👍